pkg/Library/Graph/Atoms/GraphAgent.js

```diff
@@ -17,10 +17,7 @@ async update(inputs, state, {output, isDirty, service}) {
   }
   if (inputs.publishedGraphsUrl && isDirty('publishedGraphsUrl')) {
```

Comment:
`inputs` are really intended to be unpacked in `update`. It helps keep the contracts clear. 

It's enticing to just send `input` to other methods, but I think it's better to name the properties in the long run.

Reply:
i unpacked a little more.. but to pass name properties, i need to do a more significant refactoring than I'm feeling comfortable doing as part of the other work, so i added a TODO for now
(there is also the `handleEvent` method, that needs all the _inputs_, i'll give it a try in a separate PR)